Control/lib/utils.js

```diff
@@ -40,31 +40,36 @@ function errorLogger(err, facility = 'utils') {
 }
 
 /**
-  * Util to get JSON data (parsed) from server
+  * Util to get JSON data (parsed) following a GET request
   * @param {string} host - hostname of the server
   * @param {number} port - port of the server
   * @param {string} path - path of the server request
+  * @param {JSON} options - specific request options (e.g range of accepted status code)
   * @return {Promise.<Object, Error>} JSON response
   */
-function httpGetJson(host, port, path) {
+function httpGetJson(hostname, port, path, params = undefined) {
+  const options = params ?? {
+    statusCodeMin: 200,
+    statusCodeMax: 299,
+    rejectMessage: 'Non-2xx status code: '
+  };
```

Comment:
I think it is error prone to rename params to options here

Reply:
Removed params and kept only options